contrib/scaling.asciidoc

```diff
@@ -38,15 +38,15 @@ where they have decent security, decentralization and scalability.
 == Attempts at scaling
 
 === Casper - Proof of Stake for Ethereum
-Proof of Stake as part of Casper is a relatively new consensus algorithm designed for Ethereum that has decided to split its shifting to Proof of Stake into to stages
-one called the Casper FFG aka. Vitalik's Casper and the other called Casper the friendly GHOST aka. Casper CBC/ Vlad's Casper.
+Proof of Stake is a relativly new type of consensus algorithm aims to improve/ replace the current Proof of Work consensus. The idea of Proof of Work consensus algoritm, in its simplest form, is to allow reaching consensus based on a 1 CPU 1 vote. Proof of Stake introduces a new voting power distribution idea in a 1 coin 1 vote basis.
+Proof of Stake is designed to provide blockchain with finality, strengthen the economical security, reduce centralization risks and make the protocol better for the enviorment by removing the Proof of Work energy waste.
```

Comment:
IMHO this sounds more publicity than technical.
There is an open debate on this and I do not think it is appropriate for the book to take a position.
Let's not forget that there are also shortcomings and drawbacks in PoS consensus and it's not a piece of cake.
 :-(